1189.py

```diff
@@ -0,0 +1,13 @@
+class Solution:
+    def maxNumberOfBalloons(self, text: str) -> int:
+        letter_count = Counter(text)
+        max_num = 0
+
+        while 1:
```

Comment:
I would avoid writing `while 1` in Python. Moreover, `while True` should be avoided in general, too.